src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/memory.cpp

```diff
@@ -6,10 +6,64 @@
 
 #include <vector>
 
+#include "ngraph/opsets/opset11.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "openvino/op/util/variable.hpp"
+
 using namespace LayerTestsDefinitions;
+using namespace ngraph;
+using namespace opset11;
 
 namespace {
 
+class MemoryTestGna : public MemoryTest {
+    using Super = MemoryTest;
+
+public:
+    InferenceEngine::Blob::Ptr GenerateInput(const InferenceEngine::InputInfo& info) const override {
+        // data comparison against relative threshold works correct when input data is shifted away from 0.
+        // This happens also when POT generated FQ layers are added (see below) to original model.
+        return FuncTestUtils::createAndFillBlob(info.getTensorDesc(), 5, 5);
+    }
+
+protected:
+    void CreateCommonFunc() override {
+        auto param = builder::makeParams(ngPrc, {inputShape});
+        const auto variable_info = targetDevice == CommonTestUtils::DEVICE_GPU
+                                       ? VariableInfo{Shape{inputShape}, ngPrc, "v0"}
+                                       : VariableInfo{PartialShape::dynamic(), element::dynamic, "v0"};
+        auto variable = std::make_shared<Variable>(variable_info);
+
+        auto min55 = ngraph::builder::makeConstant<float>(ngPrc, {}, {-55.001678466796875f});
```

Comment:
are this parameters fine only fo inputs data in range 5.0-10.0?

Reply:
not only, but these turn to be not optimal for inputs zero and close to zero. please see comment in MemoryTestGna::GenerateInput(..)